src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -46,6 +46,30 @@ class ActivityFeed extends React.Component<Props, State> {
 
     feedContainer: null | HTMLElement;
 
+    componentDidMount() {
+        this.scrollFeedContainerToBottom(false);
+    }
+
+    componentDidUpdate(prevProps) {
+        this.scrollFeedContainerToBottom(true, prevProps);
+    }
+
+    scrollFeedContainerToBottom = (
+        componentWasUpdated: boolean,
+        prevProps?: Props,
+    ) => {
+        if (
+            componentWasUpdated &&
+            this.feedContainer &&
+            !prevProps.feedItems &&
+            this.props.feedItems
+        ) {
+            this.feedContainer.scrollTop = this.feedContainer.scrollHeight;
+        } else if (!componentWasUpdated && this.feedContainer) {
+            this.feedContainer.scrollTop = this.feedContainer.scrollHeight;
```

Comment:
it appears these two conditions lead to the same result, could they be combined somehow?